editor/editor_node.cpp

```diff
@@ -3769,8 +3776,20 @@ Error EditorNode::load_scene(const String &p_scene, bool p_ignore_broken_deps, b
 
 	push_item(new_scene);
 
+	// Load the selected nodes.
+	if (editor_state_cf->has_section_key("editor_states", "selected_nodes")) {
+		TypedArray<NodePath> selected_node_list = editor_state_cf->get_value("editor_states", "selected_nodes", TypedArray<String>());
+
+		for (int i = 0; i < selected_node_list.size(); i++) {
+			Node *selected_node = new_scene->get_node(selected_node_list[i]);
```

Comment:
```suggestion
			Node *selected_node = new_scene->get_node_or_null(selected_node_list[i]);
```
This avoids potential errors.